gap.go

```diff
@@ -73,6 +75,17 @@ type ManufacturerDataElement struct {
 	Data []byte
 }
 
+// ServiceDataElement strores a uuid/byte-array pair used as ServiceData advertisment elements
+type ServiceDataElement struct {
+	// service uuid or company uuid
+	// The list can also be viewed here:
+	// https://bitbucket.org/bluetooth-SIG/public/src/main/assigned_numbers/company_identifiers/company_identifiers.yaml
+	// https://bitbucket.org/bluetooth-SIG/public/src/main/assigned_numbers/uuids/service_uuids.yaml
```

Comment:
I thought so as well, but indeed in case of [bthome](https://bthome.io/format/) which is my usecase, they use a company id (`0xFCD2`) with service data. Hence my comment on the function .

Reply:
It really is a service UUID, not a company ID. You can check for yourself here: https://bitbucket.org/bluetooth-SIG/public/src/413977fdb5a21c25947f3abc4d7cf26947cd0e63/assigned_numbers/uuids/member_uuids.yaml#lines-1149
Company IDs are listed here, and don't include 0xFCD2: https://bitbucket.org/bluetooth-SIG/public/src/main/assigned_numbers/company_identifiers/company_identifiers.yaml

Reply:
I will change in a subsequent PR.

Reply:
Here is a PR to fix the comment: https://github.com/tinygo-org/bluetooth/pull/275